src/main/kotlin/org/jitsi/nlj/Event.kt

```diff
@@ -45,4 +45,6 @@ class SsrcAssociationEvent(
 
 class SetMediaStreamTracksEvent(val mediaStreamTrackDescs: Array<MediaStreamTrackDesc>) : Event
 
-class SetLocalSsrcEvent(val mediaType: MediaType, val ssrc: Long) : Event
\ No newline at end of file
+class SetLocalSsrcEvent(val mediaType: MediaType, val ssrc: Long) : Event
+
+class BandwidthEstimationChangedEvent(val bandwidthBps: Long) : Event
```

Comment:
How often does the bandwidth change? At some point traversing the whole Node tree with an event will become expensive (when we know there's a single node that cares about these events)

Reply:
it's not super often, but i agree and think we may want to tweak this a bit.